backend/interview_questions/schemas.py

```diff
@@ -0,0 +1,41 @@
+from pydantic import BaseModel, Field
+
+
+class InterviewQuestionsRequest(BaseModel):
+    """Request model for generating interview questions."""
+
+    job_role: str = Field(
+        ...,
+        description="The role for which interview questions are being generated.",
+        examples=["Frontend Developer"],
+    )
+    experience_level: str = Field(
+        ...,
+        description="The experience level of the candidate.",
+        examples=["Mid"],
+    )
+    job_description: str = Field(
+        ...,
+        description="The detailed job description for the role.",
+        examples=[
+            "Develop and maintain scalable web applications using React and TypeScript."
+        ],
+    )
+
+
+class Questionanswers(BaseModel):
+    """Response model for generated technical interview questions."""
+
+    question: str = Field(..., description="The question for the interview.")
+    answer: str = Field(..., description="The answer for the interview.")
+
+
+class InterviewQuestionsResponse(BaseModel):
+    """Response model for generated interview questions."""
+
+    behavioralquestions: list[str] = Field(
+        ..., description="The list of generates behavioral interview questions."
+    )
+    Technicalquestions: list[Questionanswers] = Field(
+        ..., description="The list of   generates  technical interview questions."
+    )
```

Comment:
- Change schema name from `Questionanswers` to `InterviewQuestion` for better clarity.
- Need to convert field names in `InterviewQuestionsResponse` schema to **snake_case**.
- Remove redundant ellipsis (...).

```suggestion
class InterviewQuestion(BaseModel):
    """Model representing a single interview question with its corresponding answer."""

    question: str = Field(description="The question for the interview question.")
    answer: str = Field(description="The answer for the interview question.")


class InterviewQuestionsResponse(BaseModel):
    """Response model for generated interview questions."""

    behavioral_questions: list[str] = Field(
        description="The list of generates behavioral interview questions."
    )
    technical_questions: list[InterviewQuestion] = Field(
        description="The list of generated technical interview questions."
    )
```